api/web3server.go

```diff
@@ -175,6 +175,8 @@ func (svr *Web3Server) handlePOSTReq(req *http.Request) interface{} {
 			method = web3Req.Get("method").Value()
 		)
 		switch method {
+		case "eth_accounts":
+			res, err = svr.ethAccounts()
```

Comment:
for a method as simple as this one, why not leave the code here?
`res = []string{}`
question: empty address == []string{} ?

Reply:
https://infura.io/docs/ethereum/json-rpc/eth-accounts
`[]` should be returned.

Reply:
Adding a subfunc is friendly for testing and extension in the future.